### PR TITLE
script for creating initial files for new components

### PR DESCRIPTION
**Pull request by PreslavKozovski**
closes: https://github.com/telerik/kendo-themes/issues/2016

After this PR is merged, I will create a Wiki - "Steps of creating a new component"

### CODE REVIEW

File: scripts/new-component/create-component-files.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+const path = require('path');
+const yargs = require('yargs');
+const nunjucks = require('nunjucks');
+
+const componentName = yargs.argv._[0];
+const themes = [ 'default', 'bootstrap', 'material', 'classic' ];
+const files = fs.readdirSync(path.resolve(path.dirname(process.argv[1]) + '/templates'));
+
+nunjucks.configure(path.resolve(__dirname, './', 'templates'), { autoescape: false });
```
Comment (joneff) [inline scripts/new-component/create-component-files.js:10]: `path.resolve` adds the end slashes, so `./` is meaningless

File: scripts/new-component/create-component-files.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+if (!componentName) {
+    throw 'Pass the component name as a parameter!';
+}
+
+themes.forEach((theme) => {
+    const dist = './packages/' + theme + '/scss/';
```
Comment (joneff) [inline scripts/new-component/create-component-files.js:17]: string interpolation

File: scripts/new-component/create-component-files.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    const data = {
+        componentName: componentName,
+        theme: theme
+    };
+
+    ensureDirSync(path.join(dist + componentName));
```
Comment (joneff) [inline scripts/new-component/create-component-files.js:23]: `fs.mkdirSync(... , { recursive: true });` is sufficient.  If it fails, it fails.

**Review by joneff — CHANGES_REQUESTED** [review]
Use `path.join` and `path.resolve` consistently. Also, prefer comma, instead of `+`.

Finally, do we want `exports( ... )` or `exports(...)`

File: scripts/new-component/create-component-files.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+const path = require('path');
+const yargs = require('yargs');
+const nunjucks = require('nunjucks');
+
+const componentName = yargs.argv._[0];
```
[the remark this answers is not in the document]
Comment (PreslavKozovski) [reply scripts/new-component/create-component-files.js]: The `getArg` approach is not working when invoking a `package.json` script.

What do you think of the following approach:

```
npm run create-component-files --name=accordion
```

```
const { getEnvArg } = require('../utils');

const componentName = getEnvArg('name');
```

```
function getEnvArg(key) {
    let name = `npm_config_${key}`;
    let arg = process.env[name];

    return arg;
}
```

**Comment by PreslavKozovski** [comment]
At this point, I see no value in exporting this functionality. I believe that, for now, we are going to use it directly by running the relevant node script.

**Review by joneff — APPROVED** [review]

**Pull request merged**